jdi-light-mobile-android-native-local-template/src/test/resources/android.properties

```diff
@@ -1,16 +1,12 @@
 platformName=Android
-#deviceName=Pixel_3a_API_30_x86
 deviceName=emulator-5554
-#autoLaunch=false
 automationName=UiAutomator2
 #app=./src/main/resources/ApiDemos-debug.apk
-#app=/Users/Maksim_Meshcheriakov/Library/CloudStorage/OneDrive-EPAM/JDI/jdi-light-mobile-templates/jdi-light-mobile-android-native-local-template/src/main/resources/ApiDemos-debug.apk
 
```

Comment:
Why did you leave this commented code here? We need only necessary parameters here for Android test run.